### PR TITLE
NearbyPoint bug fix and minor fixes

### PR DESCRIPTION
**Pull request by fabiencastan**

### CODE REVIEW

File: src/cctag/Detection.cpp [hunk 1]
```diff
@@ -863,9 +863,9 @@ void cctagDetection(
         /* identification in CUDA requires a host-side nearby point struct
          * in pinned memory for safe, non-blocking memcpy.
          */
-        if( markers.size() > 60 ) {
```
Comment (fabiencastan) [inline src/cctag/Detection.cpp]: @lcalvet: is that ok for you?

**Comment by lcalvet** [comment]
The scale should not be exposed as it could be interpret as the scale of a point of interest, typically like in SIFT, SURF, etc. and could therefore be used for sorting purpose, etc. which doesn't make any sense.
The quality must not be exposed as its value its use is meaningless outside of the library. At the moment, it reveals neither the quality of extraction nor the identification one.

**Comment by fabiencastan** [comment]
1. Ok, so if the name "scale" is confusing it could be nice to rename it.
2. Is there a way to get an information about the quality of extraction?

**Comment by lcalvet** [comment]
1. The name is not confusing for contributors but will be misused by users if exposed.
2. Not a clean way yet, that is why it is not exposed.

**Comment by fabiencastan** [comment]
I removed the commit which expose the scale & quality.
@lcalvet Can you merge?

**Pull request merged**